config.py

```diff
@@ -52,6 +52,7 @@ class Config(object):
     CORS_HEADERS = 'Content-Type'
     SQLALCHEMY_DATABASE_URI =  env('DATABASE_URL', default=None)
     DATABASE_QUERY_TIMEOUT = 0.1
+    SOFT_DELETE = True
```

Comment:
Made the changes at https://github.com/shubham-padia/flask-rest-jsonapi/commit/70d4d4e0ff87f4a4bdeeb3098c05ed1d8113e61b.
@niranjan94 @SaptakS Please review again